public/components/security/roles-mapping/components/rule-editor.tsx

```diff
@@ -227,7 +253,11 @@ export const RuleEditor = ({ save, initialRule, isLoading, isReserved, internalU
     }
   };
 
-  const saveRule = () => {
+  const saveRule = async () => {
+    const isRunAS = await checkRunAsUser();
+    if (!isRunAS.data.allow_run_as) {
+      showToast('warning', 'These changes will not take effect until run_as is activated in /usr/share/kibana/data/wazuh/config/wazuh.yml and the Kibana service is restarted');
```

Comment:
the path could come from a constant, and you should add " and the Wazuh user has the `allow_run_as` as true